packages/salesforcedx-apex-replay-debugger/src/core/heapDumpService.ts

```diff
@@ -320,4 +377,19 @@ export class HeapDumpService {
   ) {
     return varContainer.name === extentValueEntry.keyDisplayValue;
   }
+
+  public isTriggerExtent(outerExtent: HeapDumpExtents) {
+    if (
+      (outerExtent.typeName === 'Boolean' ||
+        outerExtent.typeName.startsWith('List') ||
```

Comment:
Could I have a custom class called `ListA` and could this mess things up?

Reply:
You're right. The List and Map typenames should both have the "<" as part of their comparison strings. Good catch @vazexqi!